src/fheroes2/editor/editor_object_popup_window.cpp

```diff
@@ -51,10 +53,19 @@ namespace
 
             return Maps::Ground::String( tile.GetGround() );
         case MP2::OBJ_RESOURCE: {
-            const Funds funds = getFundsFromTile( tile );
-            assert( funds.GetValidItemsCount() == 1 );
+            const auto & objects = Maps::getObjectsByGroup( Maps::ObjectGroup::ADVENTURE_TREASURES );
+            for ( const auto & info : objects ) {
```

Comment:
Is the `objects` variable a bit redundant here?

```suggestion
            for ( const auto & info : Maps::getObjectsByGroup( Maps::ObjectGroup::ADVENTURE_TREASURES ) ) {
```